js/package.json

```
@@ -53,6 +54,16 @@
   },
   "jupyterlab": {
     "extension": "src/jupyterlab-plugin",
+    "outputDir": "../share/jupyter/labextensions/jupyter-threejs",
+    "sharedPackages": {
+      "@jupyter-widgets/base": {
+        "bundled": false,
+        "singleton": true
+      },
+      "threejs": {
+        "bundled": true
+      }
```

```suggestion
      }
```
The default is to bundle, so this does not have to be specified.